Sources/xcproj/PBXRezBuildPhase.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/// This is the element for the Build Carbon Resources build phase.
+final public class PBXRezBuildPhase: PBXBuildPhase {
+
+	public override var buildPhase: BuildPhase {
```

Comment:
Not sure if it's a problem with how GitHub is representing the code here but it looks like the indentation of this property is wrong.

Reply:
Whoops. Had some tabs in there. Fixed. Nice catch!